test/test_ad9081.py

```diff
@@ -13,10 +13,9 @@ def scale_field(param_set, iio_uri):
 
     dev = adi.ad9081(uri=iio_uri)
     for field in param_set:
-        if param_set[field] is not list:
-            continue
-        existing_val = getattr(dev, field)
-        param_set[field] = param_set[field][0] * len(existing_val)
+        if isinstance(param_set[field], list):
+            existing_val = getattr(dev, field)
+            param_set[field] = param_set[field][0 : len(existing_val)]
```

Comment:
This will break if len(param_set[field]) < len(existing_val).  This probably should be:
```python
param_set[field] = [param_set[field][0]]*len(existing_val)
```

Reply:
You're right. I have updated it.